src/components/Home/Home.js

```diff
@@ -36,13 +38,20 @@ const Home = () => {
   return(
     <main>
       <Switch>
+        {error && <h2 className='error'>{error}</h2>}
         <Route exact path='/' render={() => <StatePicker selectState={selectState}/> } />
-        <Route path='/parksByState' render={() => <ParkList parksByState={parksByState}/>}/>
-        <Route path='/futureVisits' render={() => <FutureVisits futureTrips={futureTrips}/>} />
+        {(!parksByState.length && !error) ?
+          <h2 className='error'>Loading park list...</h2> :
+          <Route path='/parksByState' render={() => <ParkList parksByState={parksByState}/>}/>
+        }
         <Route path='/details/:parkCode' render={({match}) => {
           const selectedPark = parksByState.find(park => park.parkCode === match.params.parkCode)
           return <ParkDetails selectedPark={selectedPark} addToFutureTrips={addToFutureTrips} />
         }}/>
+        {!futureTrips.length ?
+          <h2 className='error'>No trips saved! Return to home to start planning!</h2> :
+          <Route path='/futureVisits' render={() => <FutureVisits futureTrips={futureTrips}/>} />
+        }
```

Comment:
Oh nice, I've never seen the path="*" but that makes sense. If it's in a switch, can also just say <Route> and don't give it any path to catch everything.